rllib/env/vector_env.py

```diff
@@ -232,11 +257,38 @@ def reset_at(self, index: Optional[int] = None) -> EnvObsType:
             index = 0
         return self.envs[index].reset()
 
+    @override(VectorEnv)
+    def restart_at(self, index: Optional[int] = None) -> None:
+        if index is None:
+            index = 0
+        old_env = self.envs[index]
+        # Re-create the sub-env at the new index.
+        self.envs[index] = self.make_env(index)
+        # Try closing down the old (possibly faulty) sub-env, but ignore errors.
+        try:
+            old_env.close()
+        except Exception as e:
+            if log_once("close_sub_env"):
```

Comment:
actually why log_once?
if we are restarting a lot of envs, and some of them can't be closed, we do want to get their error msg?

Reply:
But we already got the original error message from the env in the logs, no? The one that actually caused the crash in the first place. When I'm trying to close an already crashed env, I'm actually expecting some error and wouldn't be surprised that this doesn't work. What I'm interested in is the root error that has already been thrown before at this point.